tests/mail/tests.py

```diff
@@ -305,6 +306,18 @@ def test_non_ascii_attachment_filename(self):
         payload = message.get_payload()
         self.assertEqual(payload[1].get_filename(), 'une pièce jointe.pdf')
 
+    def test_attach_file_text(self):
+        email = EmailMessage('subject', 'body', 'from@example.com', ['to@example.com'])
+        email.attach_file(os.path.join(os.path.dirname(upath(__file__)), 'file.txt'))
+        msgs_sent_num = email.send()
+        self.assertEqual(msgs_sent_num, 1)
```

Comment:
Is it possible/worthwhile to verify any of the attachment details (like the mimetype, etc.) in addition to checking that the mail is successfully sent?